api/src/users/entities/address.entity.ts

```diff
@@ -0,0 +1,62 @@
+import { Column, Entity, JoinColumn, OneToOne, PrimaryGeneratedColumn } from 'typeorm';
+import { IsString } from 'class-validator';
+import { GenericEntity } from 'src/generic/generic.entity';
+import { Country } from './country.entity';
+
+export class Point {
+  lat: number;
+  lng: number;
+
+  constructor(lat: number, lng: number) {
+    this.lat = lat;
+    this.lng = lng;
+  }
+}
+
+@Entity()
+export class Address extends GenericEntity {
+  @PrimaryGeneratedColumn({ type: 'bigint' })
+  id: number;
+
+  @Column({ name: 'address1' })
```

Comment:
We only need the name attribute when entity.attribute don't match with table.attribute.

Reply:
I agree, but I'm worried if someone renames something then we lose the table attribute?